README.md

```diff
@@ -97,6 +97,7 @@ Method                                          | Description
 `integerish($value, $message = '')`             | Check that a value casts to an integer
 `float($value, $message = '')`                  | Check that a value is a float
 `numeric($value, $message = '')`                | Check that a value is numeric
+`natural($value, $message= ''')`                | Check that a value is a non-negative integer
```

Comment:
I do not think so. 
Some standards says that natural numbers are positive integers (1,2,3..) some (like ISO 80000-2:2009) specify non-negative integers (0,1,2,3...)

There is a difference between non-negative and positive. I suggest no change here.